tidy3d/frequencies.py

```diff
@@ -0,0 +1,216 @@
+"""Frequency utilities."""
+
+import numpy as np
+
+from .constants import C_0
+
+O_BAND = (1.260, 1.360)
+E_BAND = (1.360, 1.460)
+S_BAND = (1.460, 1.530)
+C_BAND = (1.530, 1.565)
+L_BAND = (1.565, 1.625)
+U_BAND = (1.625, 1.675)
+
+
+class FrequencyUtils:
```

Comment:
let's use pydantic for this (inherit from `Tidy3DBaseModel` and define `use_wavelength` as a `pd.Field`). Just for consistency and in case we want to include something like this in GUI.